src/content/resources/concepts/layer/implementation/component-definition/_index.md

```diff
@@ -15,7 +15,7 @@ aliases:
 
 | Component Schema | Component Converters | Reference |
 |:--- |:--- |:--- |
-| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_component_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_component_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/json#converting-oscal-xml-content-to-json)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/json-index/) |
+| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_component_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_component_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/blob/main/build/README.md#converters)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/json-index/) |
 | [XML Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_component_schema.xsd) | [JSON to XML Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_component_json-to-xml-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/xml-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/xml-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/component-definition/xml-index/) |
```

Comment:
the link https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml is broken. It has not been replaced.

Reply:
I figured out how I missed this, I rebased the half of links that were incorrect (I completed the XML to JSON link replacement, but not the JSON to XML ones you cited). I rebased the original commit in the series and this was addressed.